ui/admin/setup-edit-field.php

```diff
@@ -19,7 +19,7 @@
 	}
 }
 
-$ignored_pick_objects = apply_filters( '', array( 'table' ) );
+$ignored_pick_objects = apply_filters( 'pods_ignored_pick_objects', array( 'table' ) );
```

Comment:
```suggestion
$ignored_pick_objects = array( 'table' );
```

Lets remove the filter, it's gone in 2.8 anyways